src/firefly_exchange_client/client.py

```diff
@@ -88,6 +88,10 @@ async def onboard_user(self, token:str=None):
 
         return user_auth_token
 
+    def set_uuid(self, uuid):
+        self.apis.set_uuid(uuid)
+        self.dmsApi.set_uuid(uuid)
```

Comment:
Definitely good to expose it, but I think if the user has their UUID defined in their config/constants, we should use that in the client initialization. This should be updated on lines 23 and 24

Reply:
Client initialization still first checks for UUID in self.Network and uses that uuid to initialize the API service. If not found it sets to "".

Although I see that logic is missing in DMS api initialization, adding that